.ci/azure-pipelines.yml

```diff
@@ -42,50 +45,39 @@ jobs:
         ArtifactName: 'ansys_dpf_core_wheel'
       enabled: true
 
+    - template: templates\kill-servers-windows.yml
+
     - script: |
+        cd $(System.DefaultWorkingDirectory)
         pip install -r requirements_test.txt
       displayName: Install Test Environment
-
-    - script: |
-        set THISDIR=$(System.DefaultWorkingDirectory)
-        cd tests
-        set AWP_ROOT212=%THISDIR%\server\v212
-        pytest -v --junitxml=junit/test-results.xml --cov ansys.dpf.core --cov-report=xml --reruns 3
-        
-      displayName: Test Core API
-      timeoutInMinutes: 10
-
-    - task: PublishTestResults@2
-      inputs:
-        testResultsFormat: 'JUnit'
-        testResultsFiles: 'tests/junit/test-results.xml'
-        testRunTitle: 'windowsTests'
-        publishRunAttachments: true
       condition: always()
 
     - script: |
-        set THISDIR=$(System.DefaultWorkingDirectory)
         cd $(System.DefaultWorkingDirectory)
-        set AWP_ROOT212=%THISDIR%\server\v212
-        pytest --doctest-modules --junitxml=junit/test-doctests-results.xml ansys\dpf\core
+        pytest --doctest-modules --junitxml=junit/test-doctests-results.xml ansys/dpf/core
       condition: always()
       displayName: Test API Docstrings
-      timeoutInMinutes: 5
+      timeoutInMinutes: 15
 
     - task: PublishTestResults@2
       inputs:
         testResultsFormat: 'JUnit'
-        testResultsFiles: 'junit/test-doctests-results.xml'
+        testResultsFiles: '$(System.DefaultWorkingDirectory)/junit/test-doctests-results.xml'
         testRunTitle: 'docTestsTests'
         publishRunAttachments: true
       condition: always()
 
-    - script:  |
-        set THISDIR=$(System.DefaultWorkingDirectory)
-        set AWP_ROOT212=%THISDIR%\server\v212
-        python .ci/run_examples.py
-      displayName: 'Run example scripts'
-      timeoutInMinutes: 5
+    - template: templates\kill-servers-windows.yml
+
+    - script: |
+        @echo on
+        cd $(System.DefaultWorkingDirectory)
+        rmdir /s /q .\ansys\
+      displayName: Remove local copy of ansys-dpf-core
+      condition: always()
```

Comment:
This isn't necessary as this isn't run on a self-hosted agent and all files are ephemeral on the azure machine.